testing/util.py

```diff
@@ -40,6 +40,10 @@ def cmd_output_mocked_pre_commit_home(
     return ret, out.replace('\r\n', '\n'), None
 
 
+skipif_cant_run_coursier = pytest.mark.skipif(
+    os.name == 'nt' or parse_shebang.find_executable('cs') is None,
```

Comment:
does coursier work on windows?

Reply:
just making sure you saw this :)

Reply:
Thanks for the ping :)

According to the docs Coursier should work on windows. I don't have access to windows or know much about scripting + installation on windows but I can take a pass at getting this done soon-is.

This is my current 10% time project so I won't be able to poke around with it until next week :)